types/runtime/index.d.ts

```diff
@@ -88,6 +89,11 @@ export function read(
   schema: Format,
   dateParse?: (dateString: string) => Date,
 ): object[];
+
+export type TypeInference = DataType | 'integer';
```

Comment:
Can you explain why this needs to be `| 'integer'`?

Reply:
https://github.com/vega/vega-loader#inferType

```js
vega.inferType([0,1,2,3,4])
"integer"
```